distributed/client.py

```diff
@@ -646,6 +647,21 @@ def _handle_warn(event):
         warnings.warn(msg)
 
 
+def _maybe_call_security_loader(address):
+    security_loader = dask.config.get("distributed.client.security-loader")
+    if security_loader:
```

Comment:
Nit: This isn't functionally different, but is slightly more readable. `security_loader_term` may also not be the best name to use here, I just wanted to differentiate between the fully-qualified path string and the actual function 

```suggestion
    security_loader_term = dask.config.get("distributed.client.security-loader")
    if security_loader_term:
```